.changeset/swift-kings-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-aws': patch
+---
+
+AwsOrganizationCloudAccountProcessor configuration field roleArn is deprecated in favor of new field accountId
```

Comment:
```suggestion
`AwsOrganizationCloudAccountProcessor` configuration field `roleArn` is deprecated in favor of new field `accountId`
```

this disabled the spell check for those words since they are treated as code

Reply:
Updated this